resources/yki/localisations/en.json

```diff
@@ -1,4 +1,7 @@
 {
+  "baseMessage.automaticReply": "Please, don't reply this message - it is generated automatically.",
```

Comment:
Pitäisi olla `don't reply to this message`.

Reply:
Alkuperäisissä Slackiin laitetuissa tiedostoissa tämä taitaa muutenkin olla hieman eri muodossa:
`This is an automatic message. Please do not reply to this message.`

Onko speksi päivittynyt jossain välissä?

Reply:
Juteltiin Sallan kanssa että voidaan käyttää akr:n allekirjoituksia, joissa siis tuo `Please don't reply this message.`. Mietin itsekin että siinä pitäis varmaan olla `to` välissä. Voitais vaihtaa niin tänne kuin sinnekin tämä.